src/com/aca/narekhakobyan/homework/employee/SellManager.java

```diff
@@ -0,0 +1,283 @@
+package com.aca.narekhakobyan.homework.employee;
+
+import com.aca.narekhakobyan.homework.employee.functionality.Employee;
+import com.aca.narekhakobyan.homework.employee.functionality.Management;
+import com.aca.narekhakobyan.homework.items.concepts.*;
+import com.aca.narekhakobyan.homework.items.functionality.Video;
+import com.aca.narekhakobyan.homework.items.functionality.Voice;
+import com.aca.narekhakobyan.homework.items.models.*;
+import javafx.util.Pair;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+import java.util.logging.FileHandler;
+import java.util.logging.Logger;
+
+public class SellManager implements Management, Employee {
+    private int soldAMD = 0;
+    private int stockAMD = 0;
+    private List<Gadget> gadgets;
+    private List<Gadget> nonWorkingGadget = new ArrayList<>();
+    private Logger logger = Logger.getLogger("tradeLog");
+    private String pathTradings = "src\\com\\aca\\narekhakobyan\\homework\\output\\trading.csv";
```

Comment:
don't put resource files into sources, create "resources" folder on the same level which is "src" folder and put files into it